tests/hotplug/rolloutstrategy.go

```diff
@@ -29,7 +29,7 @@ import (
 	"kubevirt.io/kubevirt/tests/testsuite"
 )
 
-var _ = Describe("[Serial][sig-compute]VM Rollout Strategy", decorators.SigCompute, Serial, func() {
+var _ = Describe("[Serial][sig-compute]VM Rollout Strategy", decorators.SigCompute, decorators.VMLiveUpdateRolloutStrategy, Serial, func() {
```

Comment:
So the `LiveUpdate` strategy is not tested in rolloutstrategy.go, but under instancetype.go only?
I have to say that this looks like an open door for mistakes in the future, as it makes sense that a `LiveUpdate` test would eventually get to this file without the relevant decorator.

I think we should either move a `LiveUpdate` strategy test to this file, create a new `Context()`, and label it with `RequiresTwoSchedulableNodes`, or change this file name and `Describe()` test to something like `"VM Stage Rollout Strategy"`.

WDYT?

Reply:
Oh, I totally read `VMRolloutStrategyLiveUpdate` instead of  `VMRolloutStrategyStage` in the file :man_facepalming: 
I agree with you! Will close this for the moment!
/close